### PR TITLE
Always on

### PR DESCRIPTION
**Pull request by shortbloke**
Adding support for setting the Power Mode to Always On.

I know this has been discussed previously in the thread, but the alternative of needing to enable 22222 and apply the change seems overly complex for something which looks to be a simple setting to apply on startup of the AddOn.

Note: This is untested, but looks to be straightforward enough.

### CODE REVIEW

**Review by adamoutler — COMMENTED** [review]
Move the initialization stuff to the area prior to the initialized message.

**Comment by adamoutler** [comment]
The next time you do a push/pull it should be properly checked.  This failed because it tried to use your credentials with my docker.  There is a separate job for docker which needs to be run only on main when I merge.

**Comment by shortbloke** [comment]
@adamoutler Can you clarify exactly what I should have done? I didn't provide any credentials and would expect the CI pipeline to use the correct credentials when building a PR to ensure compatibility ahead of emerging to Main.

**Comment by adamoutler** [comment]
Yep.  The changes were made and should take effect on next commit.

File: argonOne/run.sh [hunk 1]
```diff
@@ -93,6 +93,7 @@ t3=$(mkfloat $(jq -r '.HighRange'<options.json))
 quiet=$(jq -r '.QuietProfile'<options.json)
 createEntity=$(jq -r '."Create a Fan Speed entity in Home Assistant"' <options.json)
 logTemp=$(jq -r '."Log current temperature every 30 seconds"' <options.json)
+alwaysOn=$(jq -r '.AlwaysOn'<options.json)
```
[the remark this answers is not in the document]
Comment (shortbloke) [reply argonOne/run.sh:96]: @adamoutler I was going with the name used by the Argon team: https://github.com/Argon40Tech/Argon-ONE-i2c-Codes
Could go with:

- Always On Power Mode
- Press power button to power on 
the second would then be default true and the i2c commands reversed.

Any suggestions as to what you might prefer?

[the remark this answers is not in the document]
Comment (adamoutler) [reply argonOne/run.sh:96]: This only works on the newer V1's. The older V1's need a factory-only firmware update.  Soon after the software update, the company released v2.  The company recommends buying a new unit and won't upgrade older units.   V2's have a physical jumper and don't use this.  So it's a very small window of manufacture to which this applies.
- v1
- newer model toward end of production

So, the more I think about it, this seems like something which will cause problems with supporting it.

[the remark this answers is not in the document]
Comment (shortbloke) [reply argonOne/run.sh:96]: Fair enough. I have V1 and newer models. I'll admit I confused myself thinking it was the V1 that had the jumper and the later versions that did it via software. Sorry to waste your time. But hopefully the PR build pipeline is fixed ;-)

**Comment by shortbloke** [comment]
Limited hardware that would benefit from this PR, so opting to close.

**Pull request closed without merging**